Fix graph rendering in Safari (incorrect offset)

pjanik opened the pull request:
https://www.pivotaltracker.com/story/show/186784214

This PR fixes incorrect PixiJS layer rendering observed by @bfinzer. I hope the comments explain why it had to be fixed that way.

Generally, if any parent of the foreignObject, such as an SVG element or a G element, has any kind of translation (either via transform or x/y attributes), this breaks rendering in Safari. The only workaround I found was to apply the X/Y attributes directly to the foreignObject element. Therefore, I had to split the previous SVG parent into two G groups around the foreign object.

I've also replaced the SVG tag with a G tag and dropped setting width and height, as both adjustments seemed cleaner to me. Initially, I was unsure whether embedding SVG within SVG was allowed, but after checking, I found that it is. However, please let me know if I'm overlooking any reasons why SVG was used previously.

kswenson commented on v3/src/components/graph/components/graph.tsx:
```diff
@@ -228,12 +235,22 @@ export const Graph = observer(function Graph({graphController, graphRef, pixiPoi
             {renderPlotComponent()}
-            <foreignObject ref={pixiContainerRef} x={0} y={0} width="100%" height="100%"/>
+          </g>
+          <foreignObject ref={pixiContainerRef} width="100%" height="100%"/>
```
Not new to this PR, but I think the width and height are incorrect here -- this makes the pixi container the size of the entire component when it need only be the size of the plot area.

pjanik replied:
I've just fixed it in the last commit.

kswenson reviewed (approved):
👍 Looks good -- I noted one issue about the size of the pixi container that should be addressed at some point, but need not hold up this PR given that it wasn't introduced here, but warrants at least a TODO comment or PT story.

pjanik commented:
> 👍 Looks good -- I noted one issue about the size of the pixi container that should be addressed at some point, but need not hold up this PR given that it wasn't introduced here, but warrants at least a TODO comment or PT story.

I've addressed that and added links to bug reports that you mentioned in PT story. So, I'm merging this PR.
